core/src/world/place/location/geographical/canyon.rs

```diff
@@ -17,7 +17,7 @@ fn name(rng: &mut impl Rng) -> String {
         4 => format!("{} {} {}", adjective(rng), thing(rng), canyon_synonym(rng)),
         5 => {
             let (profession, s) = pluralize(profession(rng));
-            format!("{}{} {}", profession, s, canyon_synonym(rng))
+            format!("{}'{} {}", profession, s, canyon_synonym(rng))
```

Comment:
Here you want:

```rust
format!("{}'s {}", profession(rng), canyon_synonym(rng))
```

Using `pluralize()` when you mean `possessive()` gives you fun like "monkey" => "monk'ies". (We don't have to deal with possessive plurals right now, so no need to get fancy here. The possessive of "ibis" is "ibis's".)

Reply:
Oh! Got it. I must have forgotten about that one!

Reply:
(To be clear, there isn't actually a `possessive()` function right now. I'm just suggesting to tack on a `'s` and call it a day.)

Reply:
Yeah I realized that pretty quickly. :sweat_smile: Sounds good!